cmd/vicadmin/server.go

```diff
@@ -550,7 +553,10 @@ func deriveErrorMessage(err error) string {
 func (s *server) index(res http.ResponseWriter, req *http.Request) {
 	defer trace.End(trace.Begin(""))
 	ctx := context.Background()
-	sess, err := s.getSessionFromRequest(req)
+	sess, err := s.getSessionFromRequest(ctx, req)
+	if err != nil {
+		panic(err.Error())
```

Comment:
Is this handled and turned into a somewhat friendly error? If not is there a recovery option other than panic we can use?